worlds/hades/Items.py

```diff
@@ -0,0 +1,422 @@
+from tokenize import group
+import typing
+
+from BaseClasses import Item, ItemClassification
+from typing import Dict
+
+class ItemData(typing.NamedTuple):
+    code: typing.Optional[int]
+    progression: bool
+    event: bool = False
+    trap: bool = False
+
+
+hades_base_item_id = 666100
+
+item_table_pacts: Dict[str, ItemData] = {  
+    "HardLaborPactLevel": ItemData(hades_base_item_id, True),
+    "LastingConsequencesPactLevel": ItemData(hades_base_item_id+1, True),
+    "ConvenienceFeePactLevel": ItemData(hades_base_item_id+2, True),
+    "JurySummonsPactLevel": ItemData(hades_base_item_id+3, True),
+    "ExtremeMeasuresPactLevel": ItemData(hades_base_item_id+4, True),
+    "CalisthenicsProgramPactLevel": ItemData(hades_base_item_id+5, True),
+    "BenefitsPackagePactLevel": ItemData(hades_base_item_id+6, True),
+    "MiddleManagementPactLevel": ItemData(hades_base_item_id+7, True),
+    "UnderworldCustomsPactLevel": ItemData(hades_base_item_id+8, True),
+    "ForcedOvertimePactLevel": ItemData(hades_base_item_id+9, True),
+    "HeightenedSecurityPactLevel": ItemData(hades_base_item_id+10, True),
+    "RoutineInspectionPactLevel": ItemData(hades_base_item_id+11, True),
+    "DamageControlPactLevel": ItemData(hades_base_item_id+12, True),
+    "ApprovalProcessPactLevel": ItemData(hades_base_item_id+13, True),
+    "TightDeadlinePactLevel": ItemData(hades_base_item_id+14, True),
+    "PersonalLiabilityPactLevel": ItemData(hades_base_item_id+15, True),  
+}
+
+items_table_event: Dict[str, ItemData] = {
+    "MegVictory": ItemData(None,True,True),
+    "LernieVictory": ItemData(None,True,True),
+    "BrosVictory": ItemData(None,True,True),
```

Comment:
```suggestion
    "MegVictory": ItemData(None, True, True),
    "LernieVictory": ItemData(None, True, True),
    "BrosVictory": ItemData(None, True, True),
```
For consistency with the others